yarn-project/archiver/src/archiver/data_retrieval.ts

```diff
@@ -16,11 +16,11 @@ import {
 /**
  * Data retrieved from logs
  */
-type DataRetrieval<T> = {
+export type DataRetrieval<T> = {
   /**
-   * The next block number.
+   * Blocknumber of the last L1 block from which we obtained data.
    */
-  nextEthBlockNumber: bigint;
+  lastProcessedL1BlockNumber: bigint;
```

Comment:
Did this change because in places where consume the value we need the last processed block num and not the next one after it so I think it makes sense to pass the value around in that format.